plugins.json

```diff
@@ -2445,5 +2445,17 @@
         "requires": ">=23.07.0"
       }
     ]
+  },
+  {
+    "id": "yellowdog",
+    "releases": [
+      {
+        "version": "1.0.0",
+        "url": "https://github.com/yellowdog/nextflow-plugin-public/releases/download/1.0.0/yellowdog-1.0.0.zip",
+        "date": "2024-02-08T?14:36:00+00:00",
```

Comment:
```suggestion
        "date": "2024-02-08T14:36:00+00:00",
```

Reply:
Not sure how the question mark got here but I think it is breaking the validation rule